apps/openmw/mwgui/inventorywindow.cpp

```diff
@@ -522,47 +530,32 @@ namespace MWGui
 
             if (!force)
             {
-                std::pair<int, std::string> canEquip = ptr.getClass().canBeEquipped(ptr, player);
+                const std::pair<int, std::string>& canEquip = ptr.getClass().canBeEquipped(ptr, player);
```

Comment:
Why did you put a reference here? I can't find any difference in the generated assembly https://godbolt.org/z/icNb-q .